libraries/AP_Math/location.h

```diff
@@ -74,16 +74,6 @@ bool        location_sanitize(const struct Location &defaultLoc, struct Location
  */
 void        print_latlon(AP_HAL::BetterStream *s, int32_t lat_or_lon);
 
-// Converts from WGS84 geodetic coordinates (lat, lon, height)
-// into WGS84 Earth Centered, Earth Fixed (ECEF) coordinates
-// (X, Y, Z)
-void        wgsllh2ecef(const Vector3d &llh, Vector3d &ecef);
-
-// Converts from WGS84 Earth Centered, Earth Fixed (ECEF)
-// coordinates (X, Y, Z), into WHS84 geodetic
-// coordinates (lat, lon, height)
-void        wgsecef2llh(const Vector3d &ecef, Vector3d &llh);
```

Comment:
I know we usually don't leave unused code in master, but I would leave these in as they may be useful for future GPS drivers (I'm aware of one where this could be used in an improved driver version).